public/docs/getting_started/project.md

```diff
@@ -1,41 +1,337 @@
-# Creating a project
+# Quickstart
 
-To use most Truffle commands, you need to run them against an existing Truffle project. This section will show you how to create a new project.
+This page will take you through the basics of creating a Truffle project and deploying a smart contract to a blockchain.
 
-For those just getting started, we have many boilerplates and example applications available to you through [Truffle Boxes](/boxes). 
+## Creating a project
```

Comment:
What do we assume the user already knows at this point? Perhaps before we get started we should state some of these assumptions, and offer links (internal if we have them, external if not) to other prerequisite documentation?

To get started, I think we assume that a user knows

1. JavaScript
2. A bit about writing tests with Mocha/Chai
3. The gist of what a blockchain is
4. That contracts are programs ran by the Ethereum network
5. That Ethereum contracts are (most often) written in solidity
6. That there are different Ethereum networks (e.g. mainnet vs test chains)

Reply:
@benjamincburns Good point, a prerequisite knowledge list is a good idea. But how about we go even simpler? We have an [Ethereum Overview](http://truffleframework.com/tutorials/ethereum-overview) tutorial. Why don't we make the Ethereum tutorial the prerequisite, link to it, and make sure that all the important topics are presented there? (I believe they are, but check me.)

Reply:
Added a link to the Ethereum Overview tutorial. I think it's an improvement.